### PR TITLE
Update samples to Unity 2020.3.6f1 and MR OpenXR plugin 0.9.3

### PR DESCRIPTION
**Pull request by yl-msft**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,13 @@ such as Unity packages and images. Please install the latest git-lfs before clon
 which can be enabled [using the OpenXR Developer Tools for Windows Mixed Reality](https://docs.microsoft.com/en-us/windows/mixed-reality/develop/native/openxr-getting-started#using-preview-extensions). 
 To install this app, search for "OpenXR" in the Microsoft Store app on HoloLens 2.
 
+Recommend to run these samples on HoloLens 2 using following version of tools:
+- Visual Studio 2019, or later
+- Unity 2020.3.6f1 LTS, or later
+- Unity OpenXR plugin 1.1.1, or later
+- Mixed Reality OpenXR plugic 0.9.3, or later
```
Comment (keveleigh) [inline README.md:34]:
```suggestion
- Mixed Reality OpenXR plugin 0.9.3, or later
```

File: README.md [hunk 2]
```diff
@@ -27,6 +27,13 @@ such as Unity packages and images. Please install the latest git-lfs before clon
 which can be enabled [using the OpenXR Developer Tools for Windows Mixed Reality](https://docs.microsoft.com/en-us/windows/mixed-reality/develop/native/openxr-getting-started#using-preview-extensions). 
 To install this app, search for "OpenXR" in the Microsoft Store app on HoloLens 2.
 
+Recommend to run these samples on HoloLens 2 using following version of tools:
```
Comment (keveleigh) [inline README.md:30]: Do we want to add an MRTK version recommendation too?

**Review by keveleigh — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (yl-msft) [reply README.md]: Fixed in latest iteration.

File: README.md (hunk 2 above)
Comment (yl-msft) [reply README.md:30]: Fixed in latest iteration.

**Pull request merged**